src/main/java/org/marketdesignresearch/mechlib/core/bidder/valuefunction/XORValueFunction.java

```diff
@@ -48,7 +48,11 @@ public Bid toBid(UnaryOperator<BigDecimal> bundleBidOperator) {
     }
 
     public List<BundleValue> getOptimalBundleValueAt(Prices prices, int maxNumberOfBundles) {
-        return bundleValues.stream()
+        return this.getOptimalBundleValueAt(prices, maxNumberOfBundles, false);
+    }
```

Comment:
The return value of this function has changed. It does no longer return bundles with a negative utility. I did not find any reference where this function was used in a different way. Therefore, it is possible for me to change this.